lib/tasks/custom_erd.rake

```diff
@@ -0,0 +1,13 @@
+namespace :erd do
+  task generate: :erd_with_timestamp
+
+  task :erd_with_timestamp do
+    unless ENV['title']
+      git_sha = %x{git rev-parse HEAD}[0...8]
+      timestamp = Time.now.utc.strftime("%Y-%m-%d")
```

Comment:
I'm not sure if we should be basing the timestamp off of UTC because it could be a little confusing. For example, if someone creates the ERD at 11:00 PM EDT on August 1, Time.now.utc would be 3:00 AM UTC on August 2. The ERD would then show August 2 in the title, but from the user's perspective it's still August 1.

I think we either have to show the full timestamp including the timezone, or not show the timestamp at all. What do you think @scooter-dangle?